tests/components/recorder/test_statistics.py

```diff
@@ -57,6 +58,24 @@
 from tests.typing import RecorderInstanceContextManager, WebSocketGenerator
 
 
+@pytest.fixture
+def multiple_start_time_chunk_sizes(
```

Comment:
This fixture is also defined in `tests/components/recorder/test_history.py`, can we move it to `conftest.py` to avoid duplicated code?

Reply:
They have different patch targets. One patches history and one patches statistics